pkg/server/server.go

```diff
@@ -49,8 +49,9 @@ import (
 type ExperimentalFeatureFlag string
 
 const (
-	AuthorizationModelIDHeader = "Openfga-Authorization-Model-Id"
-	authorizationModelIDKey    = "authorization_model_id"
+	AuthorizationModelIDHeader                          = "Openfga-Authorization-Model-Id"
+	authorizationModelIDKey                             = "authorization_model_id"
+	ExperimentalEnableListUsers ExperimentalFeatureFlag = "enable-list-users"
```

Comment:
?

Reply:
This was the result of a pull from main.

Reply:
But why is this needed? Along side changes in the tests of list users 🤔 Seems like something is off on rebasing.

Reply:
👍